appcues/consumer-rules.pro

```diff
@@ -1,12 +1,19 @@
--keep class com.appcues.data.remote.**.response.** { *; }
+# generated Moshi adapaters
+-keep class com.appcues.**.*JsonAdapter { *; }
```

Comment:
This is revised to be more generic, to cover the existing `data.remote` usage, as well as the root level com.appcues.ViewElementJsonAdapter - does this seem correct?

Reply:
yea it looks correct